### ISSUE TITLE
commonjs transformation not working via JS API

### ISSUE THREAD

**Issue by cspotcode** [swc-project/swc#151]
swc does not transform ES6 import syntax into a `require()` call when invoked via the JS API. (node bindings)

## Reproduction

`npm install swc` and then:

```javascript
console.log(
  require('swc').transformSync(`
    import {lodash} from 'lodash'
  `, {
    module: {
      type: 'commonjs'
    }
  }).code
);
```

Outputs:

```
import { lodash }from 'lodash';
```

**Comment by cspotcode** [comment]
It appears that "module" transformation options can be specified via a ".swcrc" file but not programmatically via the API.

**Comment by stefanpenner** [comment]
Ya, ran into this one as well. Passing options via the JS API appears to be MIA.

**Comment by kdy1** [comment]
Fixed with swc@1.0.5

**Comment by stefanpenner** [comment]
I verified that the problem is resolved, thank you!
